### ISSUE TITLE
[suggestion]Do not take over cpu exception handler when cpu exception occurs

### ISSUE THREAD

**Issue by HarukaKaminari** [LumaTeam/Luma3DS#296]
Currently there are only exception information and cpu context with stack dump shown when cpu exceptions occured. It should be much convenient for debugging if backtrace is available.

**Comment by Subv** [comment]
You can get a backtrace by inspecting the stack dump, but doing it programatically for the ARM arch would not be easy.

**Comment by Mrrraou** [comment]
Stacktraces are pretty much unique to high level languages debugging like C. Not feasible. As @Subv said, read the stack dump to trace calls back.

**Comment by TuxSH** [comment]
Read the reg&stack dump as well as a disasm of the program you want to debug.

**Comment by HarukaKaminari** [comment]
I implemented a backtrace in my code, and registered a corresponding exception handler. But immediately I realized that Luma has taken over the cpu exception handler if UNITINFO is checked. My own handler will never be called and executed.
But switching this option to NONE will disable the exception handler as well as the ability of installing dev cias.
Is it possible to enable cpu exception handler only, with dev cia installing, but without taking over the handler?
